runtime/src/accounts_index.rs

```diff
@@ -78,6 +82,11 @@ pub struct AccountSecondaryIndexesIncludeExclude {
     pub keys: HashSet<Pubkey>,
 }
 
+#[derive(Debug, Default, Clone, Copy)]
```

Comment:
A lot of noise to introduce this struct.
This struct will contain more options as we move forward.

Reply:
Worth it! I think this is a great idea to prevent runaway function parameters from the start.